arrow/parser.py

```diff
@@ -231,8 +323,9 @@ def parse(self, datetime_string, fmt, normalize_whitespace=False):
                 f"Failed to match '{fmt}' when parsing '{datetime_string}'."
             )
 
-        parts = {}
-        for token in fmt_tokens:
+        parts: _Parts = {}
+        for token in fmt_tokens:  # type: _FORMAT_TYPE
```

Comment:
If we change the declaration of `fmt_tokens` above, this comment can probably be removed.

Reply:
Done!